config/initializers/loadbalancer_ips.rb

```diff
@@ -0,0 +1,13 @@
+aws_ips_file_path = Rails.root.join("config/aws_ips.json")
+
+# File should always exist in prod/stag but allow testing this in dev mode if file exists
+if Rails.env.production? or Rails.env.staging? or File.exists?(aws_ips_file_path)
+  json = File.read(aws_ips_file_path)
```

Comment:
if the file doesn't exist (though it should i see), this will stop the app booting right?